packages/smart-contracts/migrations/2_deploy_contracts.js

```diff
@@ -94,7 +100,21 @@ module.exports = async function(deployer) {
     // SwapToPay
     await deployer.deploy(ERC20SwapToPay, FakeSwapRouter.address, ERC20FeeProxy.address);
     console.log('SwapToPay Contract deployed: ' + ERC20SwapToPay.address);
-    
+
+    // Mock Chainlink aggregator
+    await deployer.deploy(MockChainlinkDAIUSD);
```

Comment:
I think it's time to kick-off a 3rd migration file in order to keep things separate.

Reply:
I agree. But this would mean to rearrange the order of deployment and change the address of the contracts. We should do it in a separate PR, no?